src/oracledb_datapump/files.py

```diff
@@ -852,6 +852,8 @@ def resolve_sequences_for_import(
                 resolved_files,
             )
 
+        logger.debug("Resolved dumpfiles for import: %s, ")
+
```

Comment:
This seems to be missing an actual variable

Reply:
Good catch. Linter/Pylance never catches that. See revision.